File: TestPrintstars/C++/Printingstars.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int rows, columns;
    string user_input;

    cout << "Enter Number of Rows: ";
    cin >> rows;

    cout << "Enter Number of Columns: ";
    cin >> columns;

    cin.ignore(); // Clear the newline character from previous input

    cout << "Enter your name: ";
    getline(cin, user_input);

    cout << "Hollow Box Pattern with " << rows << " rows and " << columns << " columns" << endl;

    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < columns; j++) {
            if (i == 0 || i == rows - 1 || j == 0 || j == columns - 1) {
                cout << "*  ";
            } else {
                cout << "   ";
            }
        }

        if (i == 1) {
            cout << endl;
            cout << "*  Programming Assignment 1     *"<< endl;
            cout << "*  Computer Programming I       *"<< endl;
            cout << "*     Author: " << user_input << "               *"<< endl;
            cout << "*  Due Date: Thursday, Jan. 24  *"<< endl;
        } else {
            cout << endl;
        }
    }

    return 0;
}

File: TestPrintstars/Python/PrintingStars.py
rows = int(input("Enter Number of Rows: "))
columns = int(input("Enter Number of Columns: "))
user_input =(input('Enter your name: '))


print("Hollow Box Pattern with", rows, "rows and", columns, "columns")

for i in range(0, rows):
    for j in range(0, columns):
        if(i == 0 or i == rows - 1 or j == 0 or j == columns - 1):
            print('*', end='  ')
        else:
            print(' ', end='  ')
            

    if i == 1:
        print()
        print('*','Programming Assignment 1     ', end='  *\n')
        print('*','Computer Programming I       ', end='  *\n')
        print('*','     Author: ' + user_input               , end='               *\n')
        print('*','Due Date: Thursday,Jan. 24     ',end='*\n')
    else:
        print()  

